docs/examples.md

```diff
@@ -197,6 +196,36 @@ org.gradle.configureondemand=true
 org.gradle.jvmargs=-Dfile.encoding=UTF-8
 ```
 
+## JUnit
+
+Here is a `.cirrus.yml` that (once succeeded or failed), parses and uploads JUnit reports:
+
+```yaml
+junit_test_task:
+  junit_script: <replace this comment with instructions to run the test suites>
+  always:
+    junit_result_artifacts:
+      path: "**/test-results/**/*.xml"
+      format: junit
+```
+
+If it is running on a pull request, annotations will also be displayed in-line.
+
+## Maven
+
+Official [Maven Docker images](https://hub.docker.com/_/maven/) can be used for building and testing Maven projects:
+
+```yaml
+task:
+  name: Cirrus CI
+  container:
+    image: maven:latest
+  maven_cache:
+    folder: ~/.m2
+    fingerprint_script: cat pom.xml
```

Comment:
It won't work for a project with multiple modules. I think you can simply remove `fingerprint_script`.